pkg/gktest/filter.go

```diff
@@ -0,0 +1,30 @@
+package gktest
+
+// Filter filters suites and tests to run.
+type Filter struct{}
+
+// NewFilter parses run into a Filter for selecting suites and individual tests
+// to run.
+//
+// Empty string results in a Filter which matches all tests.
+func NewFilter(run string) (Filter, error) {
+	return Filter{}, nil
+}
+
+// Suite filters the set of test suites to run by suite name and the tests
```

Comment:
I'm a little bit confused from the description and signature exactly how this will be used - perhaps it will be come clearer with an example.

Reply:
Added examples of filters.